### PR TITLE
move single_valued method of MZV to auxiliary F ring

### PR DESCRIPTION
**Pull request by fchapoton**
### 📚 Description

This is moving a little piece of code from MZV to a more proper place (auxiliary ring).

### 📝 Checklist

- [x] I have made sure that the title is self-explanatory and the description concisely explains the PR.
- [x] I have created tests covering the changes.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by videlec** [comment]
Could you be more explicit in the documentation about the mathematical meaning of `single-valued version of self`?

File: src/sage/modular/multiple_zeta_F_algebra.py [hunk 1]
```diff
@@ -721,3 +722,24 @@ def without_f2(self):
...
+
+        def single_valued(self):
+            """
+            Return the single-valued version of ``self``.
+
+            EXAMPLES::
+
+                sage: from sage.modular.multiple_zeta_F_algebra import F_algebra
+                sage: F = F_algebra(QQ)
+                sage: t = 4*F("2")+F("3")
```
Comment (videlec) [inline src/sage/modular/multiple_zeta_F_algebra.py]: why no space around `*` and `+`?

**Comment by fchapoton** [comment]
wtf ? I have first pulled develop, and now this is a mess..

**Comment by videlec** [comment]
Looks good to me.

I did not understand the purpose of merging develop.

**Comment by fchapoton** [comment]
This was my workflow, when develop was not a moving target. I was always working on top of the latest develop. I guess I have to adapt, but it's not clear to me how.

**Comment by videlec** [comment]
if you want to do that you would better do a [rebase](https://git-scm.com/docs/git-rebase) rather than a merge.

**Comment by fchapoton** [comment]
would you please "approve" the changes ?

**Review by videlec — APPROVED** [review]

**Pull request merged**